source/blog/2017-07-21-rpm-ostree-v2017.7-released.html.md

```diff
@@ -63,9 +63,9 @@ layered:
 # rpm-ostree status
 State: idle
 Deployments:
-* vmcheck
-                 Timestamp: 2017-07-24 16:07:25
-                    Commit: 20431e5b436c495e3b974bfaf231dac546ee76d3596dc97d6fca7282bdb40506
+* fedora-atomic:fedora-atomic/25/x86_64/docker-host
```

Comment:
Hmm, not sure which is best. Waiting until it makes it in a TWR makes sense, though it's nice also to give people a sneak peek and encourage them to help us test `updates-testing`. Guess it depends on the audience we're targeting. @jberkus WDYT?